app/javascript/mastodon/components/server_banner.js

```diff
@@ -40,7 +40,7 @@ class ServerBanner extends React.PureComponent {
     return (
       <div className='server-banner'>
         <div className='server-banner__introduction'>
-          <FormattedMessage id='server_banner.introduction' defaultMessage='{domain} is part of the decentralized social network powered by {mastodon}.' values={{ domain: <strong>{domain}</strong>, mastodon: <a href='https://joinmastodon.org' target='_blank'>Mastodon</a> }} />
+          <FormattedMessage id='server_banner.introduction' defaultMessage='{domain} is part of the decentralized social network powered by {mastodon}.' values={{ domain: <strong>{domain}</strong>, mastodon: <a href='https://joinmastodon.org' target='_blank' rel='noreferrer'>Mastodon</a> }} />
```

Comment:
this was specifically re-added in a recent PR since we already have a `Referrer-Policy` header that controls this globally

Reply:
I see around ~30 hits searching around the current codebase for `noreferrer`. There is a `response.headers['Referrer-Policy'] = 'origin'` in `app/controllers/concerns`, but I didn't see anything else for that header

Reply:
@Gargron I notice on the new [roadmap](https://joinmastodon.org/roadmap) there is an item `MAS-12 Remove `noreferrer` from links`. Is your preference to disable this rule and undo these changes then?